### DISCUSSION TITLE
关于高风险的末流 211 与一本院校的抉择？

### DISCUSSION THREAD

**Discussion by Illustar0** [CollegesChat/university-information#221] [Q&A]
人在湖北，602 分，位次 16685 名。相当尴尬。个人从小就比较热爱计算机，所以计算机专业大概定死的，也有信心（大概？）能学好。

目前比较钟意上海理工大学（主要地域加分，个人比较钟意上广深），但大类招生和相对较弱的计算机专业让我有些犹豫。

想够末流 211 的计算机专业也不是够不上，但会有相当大的风险，地理位置一般也不咋地，搞不好就被调剂进其他专业，后面想转计算机也肯定是难上加难。

但是考虑到现在有“就业主要看第一学历”的说法，我想知道第一学历是否真的会对就业造成这么大的影响？（尤其是在目前计算机领域趋向于饱和的情况下）

而且一本院校与 211 的资源差距真的很大么？个人对这方面确实有些缺乏了解。

还是有些摇摆的，所以想听听各位前辈的意见。

附1：肯定是有读研的打算的。这年头不读研不行啊

**Comment by Stevelikesfruits** [comment]
同湖北，632，6243名，在末流985和顶配211抉择，相当尴尬。个人从小也非常喜欢计算机。
可以挑那种专业组内比较干净的冲冲看
然后是那种有点杂的但还可以接受的，这里面仔细找找应该有合适的211可以卡在自己的成绩并且稳上计算机的
最后就是一本
这样选完后把所有选出来的大学计算机学科进行查询和排名，挑几个稍微干净的放前面
而且要是读研的话，认真学说不定有很大机会可以从一本出来去别的好学校读研，其实如果求稳的话也不用太在乎
当然冲的话有点风险，不过目前好大学对转专业要求正在放开，今年武大只要能进都会非常尊重志愿，以后转的概率很大很大
抉择还是看自己吧

附1：本来是想出来找找建议的，结果遇到了同行，也想读研，也卡着尴尬位置，同样热爱计算机

**Reply by Illustar0** [reply]
羡慕啊老哥。
确实是有的，但肯定要作出取舍，（专业水平，学校环境，分数线）三个选顶多选俩（我三色图呢？）。目前想看看能否冲一手矿大。只要进了矿大就有把握转到计算机。反正矿大转计算机时计算机机考成绩占40%，刚好有C++基础，这还卷不过真是没天理。加上矿大也是出了名的好转专业。
其实我觉得你也可以关注下院校的保研率，找个保研率更高的学校。如果拿不到保研名额，大三大四考研时就真的痛苦啊。

**Reply by Stevelikesfruits** [reply]
emmm是的，但是好像除了清北之外其他的保研率都有限

**Reply by Stevelikesfruits** [reply]
你甚至还学过一些编程，像我这种就纯对计算机感兴趣，一点没入门啊

**Reply by Illustar0** [reply]
确实，清北保研率摇摇领先（毕竟是清北），中间几乎断层了。只能说高一点是一点吧。
编程挺好入门的。emm，至少我是这么觉得的。毕竟都说兴趣是最好的老师嘛
暑假时其实也可以自己学下。大学里一般是教C或者C++？

**Reply by Illustar0** [reply]
加你了
